lib/engine/game/base.rb

```diff
@@ -2055,6 +2074,11 @@ def init_hexes(companies, corporations)
               blockers[hex] = company
             end
           end
+          abilities(company, :blocks_hexes_consent) do |ability|
```

Comment:
Why is this a separate ability rather than an option for the block_hexes ability?

Reply:
This is only a consent ability, so it's only implemented as the front end check, just like when purchasing trains for others (no real check, just a "Did X say it's ok?". It was less code to make a new ability than to edit the other one and add code to NOT do most of what the other one does in the right places. We could implement it that way for sure, this was simpler.